transitions/RotateScaleVanish.glsl

```diff
@@ -0,0 +1,40 @@
+// Author: Mark Craig
+// mrmcsoftware on github and youtube ( http://www.youtube.com/MrMcSoftware )
+// License: MIT
+
+// RotateScaleVanish Transition by Mark Craig (Copyright © 2022)
+
+uniform bool FadeInSecond; // = true
+uniform bool ReverseEffect; // = false
+uniform bool ReverseRotation; // = false
+
+#define M_PI 3.14159265358979323846
+#define _TWOPI 6.283185307179586476925286766559
+
+vec4 transition(vec2 uv)
+{
+if (progress == 0.0) { return(getFromColor(uv)); }
+else if (progress == 1.0) { return(getToColor(uv)); }
+vec2 iResolution = vec2(ratio, 1.0);
+float t;
+if (ReverseEffect) { t = 1.0 - progress; } else { t = progress; }
```

Comment:
it would help GPU to not do too much ifs. there is the `mix` helper that could help